src/pages/library/Datasets.test.js

```diff
@@ -45,7 +45,7 @@ describe('Datasets', () => {
 
   it('only shows RadX datasets when in RadX', () => {
     getEnabledBrand.mockReturnValue(brands.radX)
-    const displayedDatasets = filterAndNormalizeDatasets(
+    const displayedDatasets = filterDatasetsAndApplyTags(
```

Comment:
I think it would be better to use a name that's not specific to the method implementation, is there a name that describes the operation at a higher level?

A high level operation name (maybe `prepareDatasetsForDisplay()`) wouldn't need to change every time we change what it means to prepare a dataset for display. For example, if we remove RADx support, we could also remove the filtering. So the implementation of the method will change, but the operation is the same.